server/apiserver/apiserver.go

```diff
@@ -48,6 +47,7 @@ func (api *ApiServer) ListenAndServe() {
 	http.HandleFunc("/headers", api.Headers)
 	// http.HandleFunc("/get_image", api.HTTPGetImage)
 	http.HandleFunc("/update_pixel", api.HTTPUpdatePixel)
+	http.HandleFunc("/update_user", api.HTTPUpdateUserList)
```

Comment:
This endpoint is only for testing purpose. We might not use this endpoint in production.